pom.xml

```diff
@@ -422,6 +423,12 @@
             <version>${version.jmockit}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.picketbox</groupId>
```

Comment:
Is there no way we can solve this without adding a PicketBox dependency?

IMO I think i would prefer to fork some code rather than tie the two projects together.